test/e2e-tests.sh

```diff
@@ -40,15 +39,10 @@ export_logs_continuously "kafka-broker-dispatcher" "kafka-broker-receiver" "kafk
 
 failed=false
 
-go_test_e2e -timeout=30m ./test/... || failed=true
+go_test_e2e -timeout=50m ./test/... || failed=true
```

Comment:
Uhhhhh are you sure of that?

Reply:
I changed again to use a 2 step test since GH actions don't run the sacura job and reverted this timeout change.